.github/workflows/main.yml

```diff
@@ -57,3 +57,11 @@ jobs:
 
     - name: Building //python directory
       run: bazel build //python/...
+
+    - uses: swift-actions/setup-swift@v1
+      if: matrix.os == 'ubuntu-20.04'
+
+    - name: Building //swift directory
+      run: |
+        cd swift
+        CC=clang bazel build //...
```

Comment:
Do we need to specify clang via this variable? Usually there is a Bazel way to do that (https://bazel.build/tutorials/ccp-toolchain-config)